doc/source/reference/command-line.rst

```diff
@@ -108,13 +108,20 @@ Now that looks much better!  Note that all three axes' projections appear
 nearly indistinguishable, because of how the two spheres are located in the
 domain.  We could center our domain on one of the spheres and take a slice, as
 well.  Now let's see what the domain looks like with grids overlaid, using the
-``--show-grids`` option.:
+``--show-grids`` option:
 
 .. code-block:: bash
 
   $ yt plot --show-grids -p DD0010/moving7_0010
 
-We can now see all the grids in the field of view.
+We can now see all the grids in the field of view. If you want to
+annotate your plot with a scale bar, you can use the
+``--scale-scale-bar`` option:
```

Comment:
Looks like you missed this here but that's the only issue. I'll merge this one in manually with a fix for this. Thanks for the PR!